orchestration/scripts/poll-ingest-job.py

```diff
@@ -0,0 +1,39 @@
+import google.auth
+from google.auth.transport.requests import AuthorizedSession
+from requests.exceptions import HTTPError
+import polling
+import os
+
+credentials, project = google.auth.default(scopes=['openid', 'email', 'profile'])
+
+authed_session = AuthorizedSession(credentials)
+base_url = os.environ["API_URL"]
+headers = {"accept": "application/json"}
+
+def check_job_status(job_id: str):
+    response = authed_session.get(base_url + "jobs/" + job_id, headers=headers)
+    if response.ok:
+        return response.json()["job_status"]
+    else:
+        raise HTTPError("Bad response, got code of: {}".format(response.status_code))
+
+
+def is_done(job_id: str):
+    return check_job_status(job_id) == "succeeded"
+
+def get_job_result(job_id: str):
+    response = authed_session.get(base_url + "jobs/{}/result".format(job_id), headers=headers)
+    if response.ok:
+        return response.json()["fileId"]
+    else:
+        raise HTTPError("Bad response, got code of: {}".format(response.status_code))
+
+try:
+    polling.poll(lambda: is_done(os.environ["JOB_ID"]), step=10, timeout=300)
```

Comment:
Does the poll fail if the timeout is passed? If so, we'll probably want to make it an input variable / really max it out.

Reply:
we can set it to run indefinitely lol; do we want that or the input variable?

Reply:
Should probably do an input variable, or hard-code to like 24 hours